yaksh/test_models.py

```diff
@@ -1084,6 +1095,56 @@ def setUpClass(self):
         )
         self.mcc_based_testcase.save()
 
+        # create answerpaper2
+        self.answerpaper2 = AnswerPaper(
+            user=self.user,
+            question_paper=self.question_paper2,
+            start_time=self.start_time,
+            end_time=self.end_time,
+            user_ip=self.ip,
+            course=self.course
+        )
+
+        self.attempted_papers2 = AnswerPaper.objects.filter(
```

Comment:
Why you need to do filter?
Already self.answerpaper2 gives you the required object!